examples/pipelines_DL/Keras_DeepConvNet.yml

```
@@ -20,8 +28,8 @@ pipeline:
           from: tensorflow.keras.optimizers.legacy
           parameters:
             learning_rate: 0.0009
-      epochs: 2
-      batch_size: 1
+      epochs: 1000
+      batch_size: 64
```

```suggestion
      epochs: 2
      batch_size: 1
```
This yaml file is from an example. The documentation will be generated forever. CI computers are really simple. 😢